typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -428,12 +435,17 @@ export async function moduleCanCertainlyVerify(
   origin: ChainName,
   destination: ChainName,
 ): Promise<boolean> {
+  const originDomainId = multiProvider.tryGetDomainId(origin);
+  const destinationDomainId = multiProvider.tryGetDomainId(destination);
+  if (originDomainId === null || destinationDomainId === null) {
```

Comment:
`if (!originDomainId || !destinationDomainId)`
Direct `null` comparisons like this won't cover the case where an empty or undefined value is checked.